### ISSUE TITLE
Training on Imbalanced Custom Data

### ISSUE THREAD

**Issue by Hananali1** [microsoft/SoftTeacher#215]
Thanks a lot for this great work. I'm working on imbalanced custom data with two classes. Both the baseline model and SoftTeacher model detect only one class on the test set. What changes to the Faster RCNN model (faster_rcnn_r50_fpn.py) you would suggest mitigating the overfitting towards the major class?
Thanks
